backend/wmg/pipeline/cube_pipeline.py

```diff
@@ -49,30 +48,30 @@ def load_data_and_create_cube(
     corpus_path = f"{snapshot_path}/{snapshot_id}"
 
     integrated_corpus.run(path_to_h5ad_datasets, corpus_path, extract_data)
+
+    cube_is_invalid = False
     try:
         summary_cubes.run(corpus_path, validate_cube)
     except CubeValidationException as e:
-        cube_data_s3_path = upload_artifacts_to_s3(
-            snapshot_source_path=corpus_path,
-            snapshot_schema_version=WMG_DATA_SCHEMA_VERSION,
-            snapshot_id=snapshot_id,
-            is_snapshot_validation_successful=False,
-        )
         logger.exception(e)
-        sys.exit(f"Exiting due to cube validation failure. Failed data location: {cube_data_s3_path}")
+        cube_is_invalid = True
```

Comment:
cubes that fail validation should still be complete - previously, pipeline was exiting before making the primary filters and cell ordering json